tst/regression/test_suites/poisson_gmg/poisson_gmg.py

```diff
@@ -0,0 +1,31 @@
+# ========================================================================================
+# Parthenon performance portable AMR framework
+# Copyright(C) 2020 The Parthenon collaboration
+# Licensed under the 3-clause BSD License, see LICENSE file for details
+# ========================================================================================
+# (C) (or copyright) 2021. Triad National Security, LLC. All rights reserved.
+#
+# This program was produced under U.S. Government contract 89233218CNA000001 for Los
+# Alamos National Laboratory (LANL), which is operated by Triad National Security, LLC
+# for the U.S. Department of Energy/National Nuclear Security Administration. All rights
+# in the program are reserved by Triad National Security, LLC, and the U.S. Department
+# of Energy/National Nuclear Security Administration. The Government is granted for
+# itself and others acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+# license in this material to reproduce, prepare derivative works, distribute copies to
+# the public, perform publicly and display publicly, and to permit others to do so.
+# ========================================================================================
+
+# Modules
+import sys
+import utils.test_case
+
+# To prevent littering up imported folders with .pyc files or __pycache_ folder
+sys.dont_write_bytecode = True
+
+
+class TestCase(utils.test_case.TestCaseAbs):
+    def Prepare(self, parameters, step):
+        return parameters
+
+    def Analyse(self, parameters):
+        return True
```

Comment:
Yeah, it returns a non-zero exit status if it doesn't converge in some set number of steps.

Reply:
Do we know for sure the test fails in the CI in that case?

Reply:
I can guarantee that it fails, ask me how I know...

Reply:
I would nevertheless suggest to add a comment above that line so that people who looking into our tests as example code know why it's okay here to just return `true`.